index.html

```diff
@@ -0,0 +1,222 @@
+      <code>if ((x = y)) { /* statements here */ }</code>
+    </section>
+
+    <section class="main-section" id="while_statement">
+      <header>while statement</header>
+      <p>A while statement executes its statements as long as a specified condition evaluates to true. A while statement looks as follows:</p>
+      <code>while (condition) statement</code>
+        <p>If the condition becomes false, statement within the loop stops executing and control passes to the statement following the loop.</p>
+        <p>The condition test occurs before statement in the loop is executed. If the condition returns true, statement is executed and the condition is tested again. If the condition returns false, execution stops and control is passed to the statement following while.</p>
+        <p>To execute multiple statements, use a block statement ({ ... }) to group those statements.</p>
+        <p>Example:</p>
+        <p>The following while loop iterates as long as n is less than three:</p>
+      <code>var n = 0; var x = 0; while (n < 3) { n++; x += n; }</code>
+      <p>With each iteration, the loop increments n and adds that value to x. Therefore, x and n take on the following values:</p>
+      <ul>
+        <li>After the first pass: n = 1 and x = 1</li>
+        <li>After the second pass: n = 2 and x = 3</li>
+        <li>After the third pass: n = 3 and x = 6</li>
+      </ul>
+      <p>After completing the third pass, the condition n < 3 is no longer true, so the loop terminates.</p>
+    </section>
+
+    <section class="main-section" id="Function_declarations">
+    <header>Function declarations</header>
+    <p>A function definition (also called a function declaration, or function statement) consists of the function keyword, followed by:</p>
+    <ul>
+      <li>The name of the function.</li>
+      <li>A list of arguments to the function, enclosed in parentheses and separated by commas.</li>
+      <li>The JavaScript statements that define the function, enclosed in curly brackets, { }.</li>
+    </ul>
+
+    <p>For example, the following code defines a simple function named square:</p>
+    <code>function square(number) { return number * number; }</code>
+    <p>The function square takes one argument, called number. The function consists of one statement that says to return the argument of the function (that is, number) multiplied by itself. The return statement specifies the value returned by the function.    </p>
+    <code>return number * number;</code>
+    <p>Primitive parameters (such as a number) are passed to functions by value; the value is passed to the function, but if the function changes the value of the parameter, this change is not reflected globally or in the calling function.</p>
+  </section>
+
+  <section class="main-section" id="Reference">
+
+  <header>Reference</header>
```

Comment:
Replace your `header` element with heading element(h1, h2, h3, h4, h5, h6)

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear. eg why using the header as a substitute for your ```<hx>``` is not a good thing